core/src/test/java/org/hibernate/ogm/backendtck/embeddable/Account.java

```diff
@@ -15,8 +16,10 @@
  */
 @Entity
 public class Account {
+
 	private String login;
 	private String password;
+	@Embedded
```

Comment:
Interesting, it seems that we missed an annotation but it's not required.

Anyway, you should add this on the `getHomeAddress()` method, not the attribute.